polynote-kernel/src/main/scala/polynote/config/PolynoteConfig.scala

```diff
@@ -291,8 +296,6 @@ object PolynoteConfig {
 
     Logging.info(s"Loading configuration from $file") *> configIO
       .catchAll {
-        case _: MatchError =>
```

Comment:
This was causing issues with trying to catch on `parseFile` - looks like it was [fixed awhile ago. ](https://github.com/circe/circe-yaml/issues/50).

Reply:
It was fixed a while ago, but we probably aren't using that newer version in Polynote (because we are still supporting Scala 2.11)